production.Dockerfile

```diff
@@ -31,6 +27,18 @@ RUN apt-get update && apt-get install -y --no-install-recommends curl \
 	&& rm -rf /var/lib/apt/lists/* \
     && rm -rf frontend/dist/*.map
 
+# install dependencies but ignore any we don't need for dev environment
+RUN pip install $(grep -ivE "psycopg2" requirements.txt) --no-cache-dir --compile\
```

Comment:
Why switch around the order of this?

Reply:
Git needs to be installed before pip is ran now, as one dependency in `requirements.txt` is a `git+foo` one ([django-rest-hooks](https://github.com/zapier/django-rest-hooks) – unfortunately the version on PyPI is outdated).